fr/integrations/plugin-packs/procedures/applications-pfsense-fauxapi.md

```diff
@@ -0,0 +1,179 @@
+---
+id: applications-pfsense-fauxapi
+title: Pfsense Fauxapi
+---
+
+## Vue d'ensemble
+
+Dans son principe, FauxAPI lit simplement le fichier config.xml de pfSense, le convertit en JSON et retourne à l'appelant de l'API.
+FauxAPI fournit des interfaces API de sauvegarde et de restauration faciles qui, par défaut, stockent les sauvegardes de la configuration 
+sur toutes les opérations d'écriture de la configuration ; il est donc très facile de revenir en arrière même si l'utilisateur de l'API parvient à déployer 
+une configuration "très cassée".
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+* Applications
+* Firewall
+* Router
+* FauxAPI
```

Comment:
Should be the list of the Services templates no ?

Reply:
Done ^^